broadbean/blueprint.py

```diff
@@ -878,7 +872,7 @@ def _subelementBuilder(blueprint: BluePrint, SR: int,
             chunk = int(np.round(dur*SR))
             marker[ind:ind+chunk] = 1
 
-    output = np.array(output)  # TODO: Why is this sometimes needed?
```

Comment:
should we just remove that todo. It seems fairly obvious what is going on. Above output is defined as a list be we want to return an np array

Reply:
Agreed with this as well :)

Take a look at this change - we now create the array with `np.fromiter` with preallocation which should be slightly faster than what it previously was.